epictrack-web/src/components/workPlan/status/StatusForm.tsx

```diff
@@ -41,7 +61,63 @@ const StatusForm = () => {
           width: "100%",
         }}
         onSubmit={handleSubmit(onSubmitHandler)}
-      ></Grid>
+      >
+        <Grid item xs={4}>
+          <ETFormLabel required>Date</ETFormLabel>
+          <Controller
+            name="start_date"
+            control={control}
+            defaultValue={Moment(status?.start_date).format()}
+            render={({ field: { onChange, value }, fieldState: { error } }) => (
+              <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <DatePicker
```

Comment:
do we need a date picker for this or a textfield with type='date'?

Reply:
i used datepicker so it looks same as the rest of our date inputs